shader/camera.glsl

```diff
@@ -58,11 +58,12 @@ void get_camera_ray(
 #endif
 }
 
-vec2 get_camera_projection(in camera_data cam, vec3 world_pos)
+vec3 get_camera_projection(in camera_data cam, vec3 world_pos)
```

Comment:
Also update the other get_camera_projection for equirectangular cameras below. Maybe use the length of 't' before normalization for the Z?